MessageFormats.md

```diff
@@ -8,16 +8,45 @@ Von Agenten:
 	---|-----|------------
 	id|int|Eindeutiger Agentnummer innerhalb einer Konfigurationsdatei
 	config|String|Bezeichnung der Konfigurationsdatei
+	requires|int[]|optionale Nummern der Agent von denen dieser abh�ngig ist
+	demand|int[]|optionaler Bedarf f�r die 12 Perioden (nur 1 Agent)
 	
 	Beispiel JSON-codiert Agent 1 aus Konfigdatei "Bsp1":
 	```json
 	{ 	
 		"type"	    : "agentregistration",
 		"id"	   : 1,
-		"config" 	: "Bsp1"
+		"config" 	: "Bsp1",
+		"requires" : [2,3],
+		"demand": [1,2,3,4,5,6,7,8,9,10,11,12]
 	}
 	```
 
+-	AgentResponse:
+
+	Key|Value|Beschreibung
+	---|-----|------------
+	selection|int|gew�hlter Vorschlag
+	demands|Map<Int, Int[]>| Bedarf pro Periode f�r die entsprechende L�sung
+	cost|int|Totale Kosten f�r die gew�hlte L�sung 

```

Comment:
could you please mention explicitly that `cost` is only for debug purposes and to analyze the quality of the solution. It should be optional.